serpentTools/parsers/depmatrix.py

```diff
@@ -60,6 +63,7 @@ def __init__(self, filePath, sparse=None):
         BaseReader.__init__(self, filePath, 'depmtx')
         SparseReaderMixin.__init__(self, sparse)
         self.deltaT = None
+        self.flx = None
```

Comment:
Similar comments from #512 

1. Please add this to the `Attributes` section of the docstring, when it may exist or not, what purpose it serves, etc.
2. Please consider using the full name `self.flux`